src/ctelnet.cpp

```diff
@@ -2606,9 +2694,25 @@ bool cTelnet::loadReplay(const QString& name, QString* pErrMsg)
         }
         loadingReplay = true;
         if (mudlet::self()->replayStart()) {
-            // TODO: consider moving to a QTimeLine based system...?
-            // This initiates the replay chunk reading/processing cycle:
-            loadReplayChunk();
+            std::pair results = preparseReplayFile();
```

Comment:
```suggestion
            auto[ok, modifiedFormat] = preparseReplayFile();
```

https://skebanga.github.io/structured-bindings